packages/schematics/angular/module/index.ts

```diff
@@ -68,6 +70,74 @@ function addDeclarationToNgModule(options: ModuleOptions): Rule {
   };
 }
 
+function addRouteDeclarationToNgModule(
+  options: ModuleOptions,
+  project: WorkspaceProject,
+  routingModulePath: Path | undefined,
+): Rule {
+  return (host: Tree) => {
+    if (!options.route) {
+      return host;
+    }
+    if (options.route && !options.module) {
+      throw new Error('Module option required when creating a lazy loaded routing module.');
+    }
+
+    let path: string;
+    if (routingModulePath) {
+      path = routingModulePath as string;
+    } else {
+      path = options.module as string;
+    }
+
+    const text = host.read(path);
+    if (!text) {
+      throw new Error(`Couldn't find the module nor its routing module.`);
+    }
+
+    const ivyEnabled = isProjectUsingIvy(host, project);
+    const sourceText = text.toString('utf-8');
+    const addDeclaration = addRouteDeclarationToModule(
+      ts.createSourceFile(path, sourceText, ts.ScriptTarget.Latest, true),
+      path,
+      buildRoute(options, ivyEnabled),
+    ) as InsertChange;
+
+    const recorder = host.beginUpdate(path);
+    recorder.insertLeft(addDeclaration.pos, addDeclaration.toAdd);
+    host.commitUpdate(recorder);
+
+    return host;
+  };
+}
+
+function getRoutingModulePath(host: Tree, options: ModuleOptions): Path | undefined {
+  let path: Path | undefined;
+  const modulePath = options.module as string;
+  const routingModuleName = modulePath.split('.')[0] + '-routing';
+  const { module, ...rest } = options;
+
+  try {
+    path = findModuleFromOptions(host, { module: routingModuleName, ...rest });
+  } catch {}
+
+  return path;
+}
+
+function buildRoute(options: ModuleOptions, ivyEnabled: boolean) {
+  let loadChildren: string;
+  const modulePath = `./${options.name}/${options.name}.module`;
+  const moduleName = `${strings.classify(options.name)}Module`;
+
+  if (ivyEnabled) {
```

Comment:
The import syntax works for VE, hence we should always use this syntax

Reply:
That's my fault. Fortunately, this will decrease the size of the change this PR introduces.